File: ChatApp/app/src/main/java/top/chilfish/chatapp/helper/TimeFormat.java
package top.chilfish.chatapp.helper;

import android.annotation.SuppressLint;

import java.text.SimpleDateFormat;
import java.util.Date;

public  class TimeFormat {
  public static String toString(long time, String format) {
    Date date = new Date(time);
    @SuppressLint("SimpleDateFormat") SimpleDateFormat sdf = new SimpleDateFormat(format);
    return sdf.format(date);
  }
}

File: ChatApp/app/src/main/java/top/chilfish/chatapp/helper/TimeFormat.kt
package top.chilfish.chatapp.helper

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

object TimeFormat {
    fun formatDate(time: Long, format: String?): String {
        val date = Date(time)
        val sdf = SimpleDateFormat(format, Locale.getDefault())
        return sdf.format(date)
    }

    fun toTimestamp(dateString: String, format: String): Long {
        val sdf = SimpleDateFormat(format, Locale.getDefault())
        val date = sdf.parse(dateString)
        return date?.time ?: 0
    }
}